invenio_app_ils/views.py

```diff
@@ -39,3 +39,12 @@ def success_response(self, user):
             )
         )
         return jsonify(user_payload), 200
+
+
+logged_out_blueprint = Blueprint('logged_out', __name__)
+
+
+@logged_out_blueprint.route('/logged_out')
```

Comment:
`dash` nicer than `underscore` in the URL, WDYT?
```suggestion
@logged_out_blueprint.route('/logged-out')
```